waltz-service/src/main/java/com/khartec/waltz/service/user/RoleUtilities.java

```diff
@@ -94,6 +98,11 @@ private static Role getRequiredRoleForMeasurable(Operation op, EntityKind additi
     }
 
 
+    private static Role getRequiredRoleForMeasurableCategory(Operation op, EntityKind additionalKind) {
+        return CAPABILITY_EDITOR;
```

Comment:
does this not need the additionalKind check as per the js file?

Reply:
Will take a look, I think you're right

Reply:
fixed